test/multiMasterTest.js

```diff
@@ -20,7 +20,7 @@ describe('multi-master', function () {
     try {
       await pMap(instances, i => i.start())
     } catch (err) {
-      assert(false)
+      assert(false, err.message)
```

Comment:
Sort of commenting in a random spot, but couldn't get a successful run of the entire test suite with `npm test` locally (I could get successful test runs of individual test files using `npx mocha`). I think I was using a valid environment looking at the CI configurations: Postgres 9.5 and Node 14.

It always failed with the same cause (`sorry, too many clients already`), but the specific tests that failed were not consistent. But always something like this:
<img width="663" alt="Screen Shot 2022-06-12 at 12 05 34 PM" src="https://user-images.githubusercontent.com/18332856/173244508-8bb3bb60-7286-4c8e-8b4f-cbfd80e5369e.png">

What am I missing? FYI I was running against the official docker image `postgres:9.5`